### PR TITLE
Normalize images to [-1,1] and add reflection padding to residual blocks

### PR DESCRIPTION
**Pull request by DavidAbramian**
Normalize input images to [-1,1] range by applying `im/127.5 - 1` formula during loading. This solves several issues with the histogram  of output images, as well as training convergence.

Add reflection padding to residual blocks. This seems to have a small effect in improving edge effects.

Both these changes align this implementation with the [original pyTorch implementation](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix).

A final small change is replacing `Container` with `Network` for newer Keras versions.

### CODE REVIEW

**Comment by simontomaskarlsson** [comment]
Thank you for your pull request @DavidAbramian. I will add this together with a comment stating the issues of using this formula together with 16 bit depth images.

Regards,
Simon
